.travis.yml

```diff
@@ -1,24 +1,30 @@
 # boards are ordered by priority so that when monitoring a build developers get feedback on the important ones first.
-env:
+matrix:
 #Do not define 'global' env vars here.  They cannot be used with API builds
-    matrix:
-      - MAKETARGET=lint
-      - MAKETARGET=zip_devo8
-      - MAKETARGET=zip_devo10
-      - MAKETARGET=zip_devo12
-      - MAKETARGET=zip_devo7e
-      - MAKETARGET=zip_devof12e
-      - MAKETARGET=zip_t8sg
-      - MAKETARGET=zip_at9
-      - MAKETARGET=zip_devof7
-      - MAKETARGET=zip_win_emu_devo8
-      - MAKETARGET=zip_emu_devo8
-      - MAKETARGET=test
-#      - COMPILER=latest TARGET=devo8
+    fast_finish: true
+    include:
+      - stage: deploy
+        env: MAKETARGET="zips winzips"
+      - stage: build
+        env: MAKETARGET=lint
+      - env: MAKETARGET=zip_devo8
+      - env: MAKETARGET=zip_devo10
+      - env: MAKETARGET=zip_devo12
+      - env: MAKETARGET=zip_devo7e
+      - env: MAKETARGET=zip_devof12e
+      - env: MAKETARGET=zip_t8sg
+      - env: MAKETARGET=zip_at9
+      - env: MAKETARGET=zip_devof7
+      - env: MAKETARGET=zip_win_emu_devo8
+      - env: MAKETARGET=zip_emu_devo8
+      - env: MAKETARGET=test
+
+stages:
+    - name: deploy
+      if: type IN (api, cron)
+    - name: build
+      if: type NOT IN (api, cron)
 
-matrix:
-  fast_finish: true
-  
 dist: trusty
```

Comment:
I feel this setting slows down the ci. It needs more time to start the first target. I would prefer start quicker.